src/en/news/blog/2021/pacific-usability-advanced-install/index.md

```diff
@@ -0,0 +1,196 @@
+
+1. The following packages must be installed: ssh, python3, podman(or docker), lvm2 and chrony (or ntp)
+2. Ensure your hosts are resolvable by DNS (it’s just easier that way!)
+3. (_OPTIONAL_) ensure your sudo user account is available
+
+Ceph installation consists of a multi-step workflow that starts with a ‘bootstrap’ phase. The bootstrap process effectively brings the orchestrator into play, enabling all the other hosts and daemons to be deployed. Ceph components may now be managed as Services, governed by a [spec](https://docs.ceph.com/en/latest/cephadm/service-management/#service-specification) which introduces some default behaviours...more on that in a minute.
+
+On your first VM grab the cephadm binary and install it (docs are [here](https://docs.ceph.com/en/latest/cephadm/install/)). With cephadm in place, and the `cephadmin` sudo user account configured on each node we're ready to bootstrap the cluster.
+
+```sh
+cephadm --image myregistry:5000/ceph/ceph:v16 bootstrap \
+  --mon-ip 192.168.122.174 \
+  --ssh-private-key /home/cephadmin/.ssh/id_rsa \
+  --ssh-public-key /home/cephadmin/.ssh/id_rsa.pub \
+  --ssh-user cephadmin \
+  --skip-monitoring-stack
+```
+
+You can see we're using the `--image` override. This allows us to pull the Ceph container image from a local registry instead of reaching out to docker hub everytime!
+
+Once the bootstrap is complete, the service specifications for mon and mgr daemons use a 'count' based placement policy. This means that the orchestrator will place these daemons anywhere to meet the number of instances defined by the count. This is normally fine, but for the purposes of this post we're opting to be as pedantic as possible and control everything! To do that, we must modify the default specs for the mon and mgr, through cephadm shell.
+
+```console
+[ceph: root@cs8-1 ~]# ceph orch apply mon --placement=1
+[ceph: root@cs8-1 ~]# ceph orch apply mgr --placement=1
+```
+
+Now the mon and mgr services are limited to a single daemon, which means I can add more hosts and not have a mon or mgr pop up where I don’t want them to! Let’s add the additional cluster hosts now.
+
+```console
+[root@cs8-1 ~]# for host in cs8-2 cs8-3; do cephadm shell -- ceph orch host add ${host}; done
+```
+
+NB. there are more options available to you when adding hosts. Check out the [docs](https://docs.ceph.com/en/pacific/cephadm/host-management/), for
+more information.
+
+Here’s a picture of the target environment we're aiming for.
+
+![](images/image-1.png)
+
+Now we can assemble a single yaml file containing the specs which determine the desired state of the cluster. The root directory for ceph daemon configurations is /var/lib/ceph/\<fsid\>. This directory contains a `home` directory, which is passed through to your cephadm shell automatically... which makes it an ideal place to create our configuration files :)
+
```

Comment:
Oh, neat! I somehow didn't know about that `home` directory mount. Nice to learn new things :-)